src/ppx/mipmap.cpp

```diff
@@ -63,7 +65,15 @@ static uint64_t CalculateDataSize(uint32_t width, uint32_t height, Bitmap::Forma
     return totalSize;
 }
 
+std::vector<char> Mipmap::mStaticData = {};
+
 Mipmap::Mipmap(uint32_t width, uint32_t height, Bitmap::Format format, uint32_t levelCount)
```

Comment:
You mean within the class? Static members cannot be initialized within the header since it would initialize across all TUs including the header. They need to be initialized within a specific TU, generally the CPP file for the header.

Reply:
Oh sorry, my comment was on the wrong line, I means for the overloaded function:
```
Mipmap::Mipmap(uint32_t width, uint32_t height, Bitmap::Format format, uint32_t levelCount)
Mipmap::Mipmap(uint32_t width, uint32_t height, Bitmap::Format format, uint32_t levelCount, bool useStaticPool)
```